.github/autobuild/mac.sh

```diff
@@ -190,11 +190,15 @@ pass_artifact_to_job() {
     fi
 }
 
-appstore_submit() {
-    echo "Submitting package to AppStore Connect..."
-    # test the signature of package
-    pkgutil --check-signature "${ARTIFACT_PATH}"
+notarize() {
+    echo "Submitting artifact to AppStore Connect..."
 
+    if [[ ${ARTIFACT_PATH} == *.pkg ]]; then
+        # test the signature pkg
```

Comment:
This comment isn't clear.  What actually happens inside `pkgutil --check-signature`?  How do we know it passed or failed the "test"?

Reply:
```suggestion
        # Check if pkg is signed
```

Reply:
So if it is not signed, we get a non zero exit code and due to set -eu the script will fail.

Reply:
See change with stackoverflow link